virtualenv.py

```diff
@@ -1395,6 +1395,20 @@ def install_python(home_dir, lib_dir, inc_dir, bin_dir, site_packages, clear, sy
                 elif os.path.exists(python_dll_d_dest):
                     logger.info("Removed %s as the source does not exist", python_dll_d_dest)
                     os.unlink(python_dll_d_dest)
+
+            if not IS_PYPY:
+                # considering that on windows, python import libraries are located in
+                # the  "<root>/libs" directory, the following code will look for and
+                # copy "pythonXY.lib".
+                pythonlib_name = "python{}{}.lib".format(sys.version_info[0], sys.version_info[1])
```

Comment:
slight name tweak, ``python_lib_name`` (the logic applies to all other names too)